jormungandr-lib/src/time.rs

```diff
@@ -262,6 +273,18 @@ impl From<u64> for SystemTime {
     }
 }
 
+impl From<u64> for SecondsSinceUnixEpoch {
+    fn from(seconds_since_epoch: u64) -> Self {
```

Comment:
I'm a bit miffed about these super-easy `From` conversions, also for `SystemTime`. I think there should be a specially named method to make the code reader pause and understand the implications.